install.sh

```diff
@@ -457,151 +456,10 @@ Mac system, you can use Vagrant or another virtualization tool. See our install
-  #   of unshare(1) doesn't have a --user option.
-  #
-  # - Many people run Arch Linux, where user namespaces are disabled as a kernel option.
-  #
-  # - Many people run Debian and/or Ubuntu, where user namespaces are enabled but require a sysctl
-  #   to enable.
-  #
-  # - We used to compile a test binary, but sometimes people would install Sandstorm on cloud VMs
-  #   where there is no compiler.
-  #
-  # We export the following variable:
-  #
-  # - USERNS_CLONE_UNPRIVILEGED_NEEDS_SYSCTL_SET = yes/no
-  #
-  # Also at the end of the function, if the program is still running, unprivileged userns either
-  # works or can be made to work.
-
-  USERNS_CLONE_UNPRIVILEGED_NEEDS_SYSCTL_SET=no
-
-  # Let's run a test program that creates a user namespace. If it passes, we know that no further
-  # configuration is required.
-  #
-  # Source of this program: https://github.com/sandstorm-io/check-for-unprivileged-userns
-  local USERNS_TEST_PROGRAM="$(mktemp)"
-  printf '\x7fELF\x02\x01\x01\x00kmc!!!\n\x00\x02\x00>\x00\x01\x00\x00\x00x\x00@\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00@\x008\x00\x01\x00\x00\x00\x00\x00\x00\x00\x01\x00\x00\x00\x07\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x00\x00@\x00\x00\x00\x00\x00\x9d\x00\x00\x00\x00\x00\x00\x00\x9d\x00\x00\x00\x00\x00\x00\x00\x00\x10\x00\x00\x00\x00\x00\x00\xb8i\x00\x00\x00\xbf\xfe\xff\x00\x00\x0f\x05\xb8\x10\x01\x00\x00\xbf\x00\x00\x00\x10\x0f\x05H\x89\xc1\xb8<\x00\x00\x00H\x89\xcf\x0f\x05' > "$USERNS_TEST_PROGRAM"
-  chmod a+x "$USERNS_TEST_PROGRAM"
-  if "$USERNS_TEST_PROGRAM" ; then
-    return  # success!
-  fi
-
-  # Clean up after ourselves.
-  rm -f "$USERNS_TEST_PROGRAM"
-
-  # If we're still here, the test program failed.
-
-  # Let's see if there's a Debian-style sysctl. If not, bail out because we have no idea how to help
-  # the user.
-  if [ ! -e /proc/sys/kernel/unprivileged_userns_clone ]; then
-    fail "E_NO_USERNS" "Your kernel does not appear to be compiled with" \
-         "support for unprivileged user namespaces (CONFIG_USER_NS=y), or something else is" \
-         "preventing creation of user namespaces. This feature is critical for sandboxing." \
-         "Read more at: " \
-         "https://docs.sandstorm.io/en/latest/administering/faq/#enabling-user-namespaces"
-  fi
-
-  # If the test binary failed, but the sysctl is set to 1, then allow the install to
-  # continue. Perhaps the user has /tmp mounted noexec or something like that.
-  if [ "$(</proc/sys/kernel/unprivileged_userns_clone)" = "1" ] ; then
-    return
-  fi
-
-  # If /proc/sys is mounted read-only, then they won't be able to run sysctl with our
-  # help. Similarly if there's no /etc/sysctl.conf we won't be able to help.
-  #
-  # The /proc/sys issue applies to Docker containers and probably other sorts of containers, too.
```

Comment:
Given this change, I should re-test in Docker and update our docs. I added a note to the docs issue #2645.